### ISSUE TITLE
LMDB file for ChairsSDHom dataset?

### ISSUE THREAD

**Issue by vivasvan1** [NVlabs/PWC-Net#68]
beginner here.
I want to fine-tune PWC-Net on ChairsSDHom dataset.
I suppose I have to convert ChairsSDHom to lmdb format but I am not sure how?
And where can I set the learning rate for fine-tuning purpose?

**Comment by liruoteng** [comment]
Hi: 
You can modify for your own purpose from a script here:  [create_lmdb.py](https://gist.github.com/liruoteng/c29f62adeb1a6e365046e239bdc430a5)
To make that script run, you probably need dependency files `flowlib.py` and `pfm.py` from : [optical flow toolkit ](https://github.com/liruoteng/OpticalFlowToolkit/tree/master/lib)

**Comment by vivasvan1** [comment]
Thank you so much liruoteng!
I have created the lmdb file.
Can you guide how to load the FlyingChairs pretrained model before starting the training?

**Comment by vivasvan1** [comment]
A gentle reminder in case you missed this issue

**Comment by fjchange** [comment]
@vivasvan1 Have you finished the process of finetuning PWC on ChairSDHom? Can you share?

**Comment by vivasvan1** [comment]
No I did not.
i got more interested in flowNet networks so i started testing on that.

**Comment by fjchange** [comment]
> No I did not.
> i got more interested in flowNet networks so i started testing on that.

I think the subnet of flownet2, flownetSD may be more suitable as it is trained on ChairSDHom.
